corehq/apps/api/resources/v0_4.py

```diff
@@ -120,6 +120,7 @@ def obj_get_list(self, bundle, domain, **kwargs):
 
         def wrapper(doc):
             if doc['doc_type'] in xform_doc_types:
+                doc.pop('user_id', None)  # needed when xform is stored in sql
                 return xform_doc_types[doc['doc_type']].wrap(doc)
```

Comment:
One option is to do a non-backwards compatible v0_6 using Django Rest
Framework (just for forms and cases). Also looks like tasypie has support
for non ORM data:
http://django-tastypie.readthedocs.io/en/latest/non_orm_data_sources.html.

Also seems like if we can avoid wrapping them at all that would be a big
win. And I don't really see why we shouldn't be able to. We've already put
effort into standardizing the ES data format so we should just be able to
serve a transformed version of that (cases may require more transformation
than forms if we want to stick with the current API format).